checkov/bicep/parser.py

```diff
@@ -46,3 +53,38 @@ def get_files_definitions(
         logging.info(f"[bicep] successfully parsed {len(definitions)} files")
 
         return definitions, definitions_raw, parsing_errors
+
+    def get_folder_definitions(
+            self, root_folder: str, excluded_paths: Optional[list[str]]
+    ) -> tuple[dict[Path, BicepJson], dict[Path, list[tuple[int, str]]], list[str]]:
+        files_list: set[Path] = set()
+        for root, d_names, f_names in os.walk(root_folder):
+            filter_ignored_paths(root, d_names, excluded_paths)
+            filter_ignored_paths(root, f_names, excluded_paths)
+            for file in f_names:
+                file_ending = os.path.splitext(file)[1]
+                if file_ending in BICEP_POSSIBLE_ENDINGS:
+                    full_path = os.path.join(root, file)
+                    files_list.add(Path(full_path))
+
+        return self.get_files_definitions(files_list)
+
+    def create_definitions(
+            self, root_folder: str,
+            files: Optional["Collection[Path]"] = None,
+            runner_filter: RunnerFilter = RunnerFilter(),
+    ) -> tuple[dict[str, DictNode], dict[str, list[tuple[int, str]]]]:
```

Comment:
```suggestion
    ) -> tuple[dict[Path, BicepJson], dict[Path, list[tuple[int, str]]]]:
```
probably this more correct